libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -105,6 +105,18 @@ class AC_AttitudeControl {
     // Sets and saves the yaw acceleration limit in centidegrees/s/s
     void save_accel_yaw_max(float accel_yaw_max) { _accel_yaw_max.set_and_save(accel_yaw_max); }
 
+    // get the roll angular velocity limit in radians/s
+    float get_ang_vel_roll_max() const { return _ang_vel_roll_max; }
+
+    // get the pitch angular velocity limit in radians/s
+    float get_ang_vel_pitch_max() const { return _ang_vel_pitch_max; }
+
+    // get the yaw angular velocity limit in radians/s
+    float get_ang_vel_yaw_max() const { return _ang_vel_yaw_max; }
+
+    // set the rate control input smoothing time constant
```

Comment:
minor correction.  comment should say "get" instead of "set"

Reply:
fixed in a follow up commit, thanks!